docs/pipelineruns.md

```diff
@@ -791,9 +791,8 @@ spec:
             image: busybox
             command: ["/bin/sh", "-c"]
             args:
-              - echo
-              # - $(params.uid)
-              - $(tasks.add-uid.results.uid)
+              - echo $(tasks.add-uid.results.uid)
```

Comment:
The previous method, although it would replace with specific values, could not be outputted by echo, leading to the misconception that it was not effective.